specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualNetworkGateway.json

```diff
@@ -362,6 +362,49 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/resetvpnclientsharedkey": {
```

Comment:
Do we have this operations in other API versions as well? 2018-07-01?

Reply:
No, this is from 2018-06-01 onwards

Reply:
@henry416 in this case you might need to port the changes to 2018-07-01 as well. Also 2018-08-01 (working branch for that version is `Network-September-Release`)

Reply:
@number213 If that is the case, I will open another PR for the 2018-08-01 version instead